### ISSUE TITLE
there is an error on tutorial

### ISSUE THREAD

**Issue by dingli06** [pytorch/tutorials#1799]
link is here `https://pytorch.org/tutorials/beginner/basics/transforms_tutorial.html`
code is here
`import torch`
`from torchvision import datasets`
`from torchvision.transforms import ToTensor, Lambda`

`ds = datasets.FashionMNIST(`
   ` root="data",`
    `train=True,`
   ` download=True,`
  `  transform=ToTensor(),`
 `   target_transform=Lambda(lambda y: torch.zeros(10, dtype=torch.float).scatter_(0, torch.tensor(y), value=1))`
`)`

I find that **transform and taget_transform** not effect data and labels,it remains original digit value [0,255] not [0,1] and it's label never change to be a hot-encoder platform

cc @suraj813

**Comment by spzala** [comment]
@dingli06 I couldn't reproduce. For example, `Shape of labels: torch.Size([1, 10])` is as expected and as explained for Lambda usage, https://pytorch.org/tutorials/beginner/basics/transforms_tutorial.html#lambda-transforms
Considering the issue is over a year old, I am curious if you can still reproduce or we should close it. Thanks!

**Comment by svekars** [comment]
Closing since it looks like others could not reproduce. Feel free to reopen if this is still an issue.
